webpack/assets/javascripts/bundle.js

```diff
@@ -8,6 +8,8 @@ require('expose?JsDiff!diff');
 require('./bundle_flot');
 require('./bundle_select2');
 require('./bundle_datatables');
+var Turbolinks = require('turbolinks');
```

Comment:
Is the `Turbolinks` variable needed in any of the older js? if so, please use the expose loader (see examples a few lines above)